wrapanapi/systems/rhevm.py

```diff
@@ -166,14 +166,12 @@ def _get_state(self):
     @property
     def ip(self):
         """
-        Returns IP address of the VM/instance
+        Returns IPv4 address of the VM/instance
         """
-        rep_dev_service = self.api.reported_devices_service()
-        try:
-            first = rep_dev_service.list()[0]
-            return first.ips[0].address
-        except IndexError:
-            return None
+        for ip in self.all_ips:
+            if ':' not in ip:
```

Comment:
I think that this should be more specific in only filtering out `fe80::` link-local addresses